### PR TITLE
ESbuild single image fixes + nodemon changes

### PR DESCRIPTION
**Pull request by mike12345567**
## Description
Fixes the single image build, as well as fixing the nodemon process for the server.

### CODE REVIEW

**Review by aptkingston — APPROVED** [review]

File: hosting/single/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 # install node-gyp dependencies
 RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends apt-utils cron g++ make python
 
 # add pin script
 WORKDIR /
-ADD scripts/pinVersions.js scripts/cleanup.sh ./
+ADD scripts/cleanup.sh ./
 RUN chmod +x /cleanup.sh
 
 # build server
```
Comment (adrinr) [inline hosting/single/Dockerfile:11]: We will be already using the built version, right? If so, this comment needs to be removed and we can change this to be `yarn install --frozen-lockfile --production=true`

File: hosting/single/Dockerfile [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 WORKDIR /app
 ADD packages/server .
-RUN node /pinVersions.js && yarn && yarn build && /cleanup.sh
+RUN yarn && /cleanup.sh
 
 # build worker
 WORKDIR /worker
 ADD packages/worker .
-RUN node /pinVersions.js && yarn && yarn build && /cleanup.sh
+RUN yarn && /cleanup.sh
```
Comment (adrinr) [inline hosting/single/Dockerfile:19]: Same here

File: hosting/single/Dockerfile (hunk 1 above)
Comment (adrinr) [reply hosting/single/Dockerfile:11]: Actually, we need to do the same to the worker/server dockerfile

Comment (mike12345567) [reply hosting/single/Dockerfile:11]: Fixed - added for server/worker as well.

**Review by adrinr — APPROVED** [review]

**Pull request merged**